.travis-mirage.sh

```diff
@@ -0,0 +1,57 @@
+## mirage setup
+
+set -ex
+
+## fetch+execute the OCaml/opam setup script
+wget https://raw.githubusercontent.com/mor1/ocaml-travisci-skeleton/master/.travis-ocaml.sh
+sh .travis-ocaml.sh
+
+## install mirage
+export OPAMYES=1
+eval $(opam config env)
+opam install mirage
+
+## copy travis `config.ml` in place
+cp .travis-config.ml src/config.ml
```

Comment:
This seems pretty specific. Why not using `src/config.ml` directly? Or maybe use `MIRAGE_CONFIG_FILE` to let the user pick choose which file to choose?